src/app/business/delfin/storages/storages.component.ts

```diff
@@ -281,7 +281,44 @@ export class StoragesComponent implements OnInit {
     }
     getAllActiveAlerts(){
         this.ds.getAllAlerts().subscribe((res)=>{
-            this.allActiveAlerts = res.json().alerts;
+            let alertsFromAlertManager = res.json().data;
```

Comment:
Similar to above comment

Reply:
This is fixed. Moved the alert data model to the consts file.